src/mbgl/sprite/sprite_parser.cpp

```diff
@@ -24,7 +25,10 @@ SpriteImagePtr createSpriteImage(const PremultipliedImage& image,
         ratio <= 0 || ratio > 10 ||
         srcX >= image.size.width || srcY >= image.size.height ||
         srcX + width > image.size.width || srcY + height > image.size.height) {
-        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics");
+        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics: %" PRIu32 "x%" PRIu32 "@%" PRIu32 ",%" PRIu32 " in %" PRIu32 "x%" PRIu32 "@%sx sprite",
```

Comment:
Got this working by reading `inttypes.h` and realizing that `PRIu32` just expands to `u`